pyccel/stdlib/STC_Extensions/List_extensions.h

```diff
@@ -18,5 +19,28 @@ static inline i_key _c_MEMB(_pull_elem)(Self* self, intptr_t pop_idx) {
     }
     return *(itr.ref); // Return the element that is being popped.
 }
+#if defined(i_use_cmp)
+// Function to get the minimum element from the vector
+static inline i_key _c_MEMB(_min)(const Self* self) {
+    i_key min_val = *_c_MEMB(_front)(self);
+    c_foreach(it, Self, *self) {
+        if (*(it.ref) < min_val) {
```

Comment:
Shouldn't you use `i_less` to ensure that this keeps working if we add complex support later?

Reply:
fixed!